workspaces/rstudio/Dockerfile

```diff
@@ -48,6 +49,9 @@ COPY rstudio-nginx.conf /etc/nginx/nginx.conf
 # at startup.
 RUN mkdir -p /var/pl-var && \
     chown -R 1001:1001 /var/lib/nginx /var/log/nginx /var/pl-var /etc/nginx/nginx.conf
+# Make sure that rstudio-server runs as our normal user to prevent startup
+# errors
```

Comment:
I would wrap that one line. I also find a few of other comment lines 'too short'.  But I guess that is between me, you, and your VC Code settings.  :grinning: 

The new build works, after a stumbly start (per slack discussion):

![image](https://github.com/PrairieLearn/PrairieLearn/assets/673121/20ac0294-28ef-49b8-ad90-713a8a96f488)

Reply:
I usually try to keep lines around 80 characters if I can, for historical reasons. Maybe I should standardize on 100 from now on.

Reply:
Same. I saw a decent editorconfig snippet the other day that had both decent indentation (not the current overly 'vertical' style that is so fashionable these days) and I always sit in front of a wide screen monitor.  In Markdown I usually tell it to fold after 100 cols, which is a little less that one half-screen for me.

But I was just sniping because snoping about style is cheap and gentle fun.  This is your file, so your style goes (using both a singular 'you' for you as author and 'all of you' for the org).